packages/manager/apps/pci-load-balancer/src/api/data/pool.ts

```diff
@@ -5,6 +5,7 @@ import {
   TProtocol,
 } from './load-balancer';
 
+// <editor-fold desc="Types">
```

Comment:
nitpick : could you please remove editor specific comments as they are not meaningful of the actual code